### DISCUSSION TITLE
Welcome to python-youtube Discussions!

### DISCUSSION THREAD

**Discussion by MerleLiuKun** [sns-sdks/python-youtube#152] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thuibr** [comment]
Hey there, great package! Just wondering, does this package have the ability to track your quota and time out when necessary?

**Reply by MerleLiuKun** [reply]
Sorry there not have an api to get your current quota.

As for me, We are tracking the quta outside. and manage different keys.
